### PR TITLE
Persistent state for advent calendar

### PR DESCRIPTION
**Pull request by kee-reel**

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+  'Serbian': false,
+  'Diving': false,
+  'Performance poi': false,
+  'Shibari': false,
+  'Model figures': false,
+  'Make-up art': false,
+  'Singing': false,
+  'Dancing': false,
+  'Gaming': false,
+  'Pet-project': false,
```
Comment (SashaGraves) [inline src/constants.js]: Is it pet-project what we are doing here?

File: src/constants.js [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+  'Yachting': false,
+  'Animation': false,
+  'Hang out with friends': false,
```
Comment (SashaGraves) [inline src/constants.js]: Well hell yeah!

File: src/constants.js [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
+  'Street-art': false,
+  'Slackline': false,
+  'Roller-scates': false,
```
Comment (SashaGraves) [inline src/constants.js]: *s**k**ates probably

File: src/constants.js [hunk 4]
```diff
@@ -1,30 +1,31 @@
...
+  'Via Ferrata': false,
+  'Camping': false,
+  'Woodworking': false,
+  'Gardening': false,
+  'Self-sustained terrarium': false,
```
Comment (SashaGraves) [inline src/constants.js]: self-sustain**ing** ?

**Review by SashaGraves — APPROVED** [review]
Looks great! Just few comments

File: src/constants.js (hunk 1 above)
Comment (kee-reel) [reply src/constants.js]: Looks like it!
Goes into flipped state then 🤟

File: src/constants.js (hunk 2 above)
Comment (kee-reel) [reply src/constants.js]: Doesn't look like a hobby, but still awesome :D
Should we flip it?

File: src/constants.js (hunk 3 above)
Comment (kee-reel) [reply src/constants.js]: Will do!

File: src/constants.js (hunk 2 above)
Comment (SashaGraves) [reply src/constants.js]: Well, I know few people who do not hang with friends at all, sooo probably it counts as one! Also, this is a good reminder to do it sometimes, i guess xD

File: src/constants.js (hunk 4 above)
Comment (kee-reel) [reply src/constants.js]: Sheesh, seems that you're right (googled for the term)
I've took it off the top of my head back then >_<

Comment (SashaGraves) [reply src/constants.js]: Yeah, no worries, that's what code reviews are for 👍

**Pull request merged**